packages/env/lib/env.js

```diff
@@ -372,6 +372,15 @@ async function configureWordPress( environment, config ) {
 		options
 	);
 
+	// Set wp-config.php values.
+	for ( const [ key, value ] of Object.entries( config.config ) ) {
+		await dockerCompose.run(
+			environment === 'development' ? 'cli' : 'tests-cli',
+			[ 'wp', 'config', 'set', key, value, '--raw' ],
```

Comment:
I tested setting a non-boolean config by putting `"WP_MAX_MEMORY_LIMIT": "512M"` in my `.wp-env.override.json` and unfortunately it added this invalid syntax to my `wp-config.php`:

```php
define( 'WP_MAX_MEMORY_LIMIT', 512M );
```

Note the lack of `''`s.

I'm not 100% sure how to fix this. Maybe:

```js
[ 'wp', 'config', 'set', key, JSON.stringify( value ), '--raw' ],
```

Or, alternatively we could not pass `--raw` if `typeof value === 'string'`.

Reply:
I suspected this.

>I'm not 100% sure how to fix this. Maybe:

That would make `false` a truthy string.

>Or, alternatively we could not pass --raw if typeof value === 'string'.

That works, done.

Reply:
> That would make false a truthy string.

Yes but WP CLI would insert `'false'` into the `define()` as `false` because we're specifying `--raw`.

What you did requires less mental gymnastics—let's go with it! 👍

Reply:
Oh, and strings would get extra quotes, you're right, I missed that.